src/TitleReceive/TitleReceiveList.js

```diff
@@ -104,7 +115,7 @@ export const TitleReceiveList = ({
       };
     },
     // eslint-disable-next-line react-hooks/exhaustive-deps
-    [],
+    [poLineLocationIds],
```

Comment:
what is about `locations`? it's also could be changed.

Reply:
please use `// eslint-disable-next-line react-hooks/exhaustive-deps` carefully, since it can easily hide error for you.

Reply:
Thanks, missed it